api/WheelEvent.json

```diff
@@ -254,45 +206,45 @@
           }
         }
       },
-      "DOM_DELTA_LINE": {
+      "deltaX": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WheelEvent/DOM_DELTA_LINE",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WheelEvent/deltaX",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true
             },
             "chrome": {
-              "version_added": null
+              "version_added": "31"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": true
             },
             "edge": {
-              "version_added": null
+              "version_added": true
             },
             "edge_mobile": {
-              "version_added": null
+              "version_added": true
             },
             "firefox": {
-              "version_added": null
+              "version_added": "17"
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "17"
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
```

Comment:
Assume we're not adding a `notes` field here for `ie`?

Reply:
Thanks for catching it! I added it